packages/pypy3

```diff
@@ -0,0 +1,66 @@
+require 'package'
+
+class Pypy3 < Package
+  description 'pypy is fast, drop-in replacement of cpython that can execute code over 4 times faster.'
+  homepage 'https://www.pypy.org/'
+  version '7.3.3'
+  compatibility 'x86_64, x86, aarch64'
+  source_url 'https://downloads.python.org/pypy/pypy3.7-v7.3.3-src.tar.bz2'
+  source_sha256 'f6c96401f76331e474cca2d14437eb3b2f68a0f27220a6dcbc537445fe9d5b78'
+
+  def self.prebuild
+			# Delete original tarball contents
+			system "rm * -rf"
+      # Download prebuilt tarball
+      if `uname -m` == "x86_64\n"
+          system "curl -#LO https://downloads.python.org/pypy/pypy3.7-v7.3.3-linux64.tar.bz2"
+          if `sha256sum pypy3.7-v7.3.3-linux64.tar.bz2` != "37e2804c4661c86c857d709d28c7de716b000d31e89766599fdf5a98928b7096  pypy3.7-v7.3.3-linux64.tar.bz2\n"
+              puts "Checksum mismatch. :/ Try again.".red
+              system "exit 1"
+          end
+      elsif `uname -m` == "x86\n"
```

Comment:
This will never be true.  It should be `elsif `uname -m` == "i686\n"`.  `case ARCH ... when ... else ... end` is a bit more elegant.